src/data/mainnet/tokens-info.json

```diff
@@ -363,7 +362,8 @@
     "decimals": 18,
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/cBTC.png",
     "name": "Wrapped Bitcoin",
-    "symbol": "BTC"
+    "symbol": "BTC",
+    "isSwappable": true
```

Comment:
This is fine, since this btc uses `18` as decimals, so we can enable it in older versions as well.